dev-python/dbutils/dbutils-1.3.ebuild

```diff
@@ -0,0 +1,43 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+PYTHON_COMPAT=( python3_{6,7} )
+
+inherit distutils-r1
+
+MY_PN="DBUtils"
+MY_P="${MY_PN}-${PV}"
+
+DESCRIPTION="Database connections for multi-threaded environments"
+HOMEPAGE="
+	https://cito.github.io/DBUtils/
+	https://github.com/Cito/DBUtils
+	https://pypi.org/project/DBUtils/
+"
+SRC_URI="mirror://pypi/${P:0:1}/${PN}/${MY_P}.tar.gz"
+
+LICENSE="OSL-2.0"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="doc"
+
+DEPEND=""
+RDEPEND=""
+
+S="${WORKDIR}/${MY_P}"
+
+distutils_enable_tests nose
+
+python_prepare_all() {
+	sed -i -e "s/, 'DBUtils.Tests'//" \
+		-e "/package_data=/d" \
+		setup.py || die "sed failed"
+	distutils-r1_python_prepare_all
+}
+
+python_install_all() {
+	use doc && local HTML_DOCS=( "${S}/${MY_PN}"/Docs/. )
```

Comment:
How many files will this include? I think you should just install them unconditionally, allowing removal of *doc* USE with it.
Since *doc* is not needed to build these files.